vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -42,21 +43,30 @@
 ElementTree.register_namespace("mstts", NAMESPACES["mstts"])
 
 
+class WordBoundaryEvent(TypedDict):
+    text: str
+    text_offset: int
+    audio_offset: float
+    boundary_type: speechsdk.SpeechSynthesisBoundaryType
```

Comment:
For typing within the message up to fcn

Reply:
https://learn.microsoft.com/en-us/javascript/api/microsoft-cognitiveservices-speech-sdk/speechsynthesisboundarytype?view=azure-node-latest

Reply:
sounds good, but is it just me or that the most useless piece of documentation ever? is that the best azure can do?

Reply:
Oh yeah its terrible